plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/flareChat/AwsServerCapabilitiesProvider.kt

```diff
@@ -52,10 +53,33 @@ data class QuickActions(
 )
 
 data class QuickActionsCommandGroups(
-    val commands: List<Command>,
+    val commands: List<QuickActionCommand>,
 )
 
-data class Command(
-    val command: String,
-    val description: String?,
+open class QuickActionCommand(
+    open val command: String,
+    open val description: String,
+    open val placeholder: String? = null,
+    open val icon: IconType? = null,
+)
+
+data class ContextCommand(
+    val id: String?,
+    val route: List<String>?,
+    val label: String?,
+    val children: ContextCommandGroup?,
+    override val command: String,
+    override val description: String,
```

Comment:
The description should be nullable too. Prompts have description as null

Reply:
This will break prompts

Reply:
ah, make sense.